internal/batches/service/service.go

```diff
@@ -226,7 +241,30 @@ func (svc *Service) BuildTasks(ctx context.Context, repos []*graphql.Repository,
 
 	var tasks []*executor.Task
 
-	attr := &executor.BatchChangeAttributes{Name: spec.Name, Description: spec.Description}
+	// TODO(mrnugget): Factor this out
```

Comment:
I'm assuming you are still intending TODO this before merging? Or did you already factor it out and forgot this comment? (I don't know what the "this" refers to.)

Reply:
Yeah. I want to clean this up and possibly do some architectural changes. The PR wasn't meant as "ready for review" or "ready to merge" yet, just as a showcase for the functionality.